### DISCUSSION TITLE
Genuine Boards?

### DISCUSSION THREAD

**Discussion by netmgr** [erichelgeson/BlueSCSI-v1#225] [Q&A]
Can I swap the knockoff bluepill for a genuine development board?
Something like the STM32 Nucleo-32, NUCLEO-F042K6

Are they the same footprint?

**Comment by erichelgeson** [comment]
These both look like they only have 32kb of flash. A very brief glance - the pinout does not look the same. It would take some effort to adapt to these boards, especially the limited flash size.
